terraform/applications/careers/static.tf

```diff
@@ -0,0 +1,189 @@
+# all the relevant infra for the static site
+resource "aws_s3_bucket" "logs" {
+  bucket = "mozilla-careers-logs-783633885093"
+  acl    = "log-delivery-write"
+}
+
+resource "aws_s3_bucket" "mozilla-careers" {
+  bucket = local.bucket_name
+  acl    = "log-delivery-write"
+
+  force_destroy = true
```

Comment:
This allows us to delete a bucket containing objects, which I guess is a valid choice, but I can't work out from the PR why this is valuable to us here.

Reply:
Just makes writing the terraform a touch easier, since you can delete the bucket with a simple tf destroy.  I added a comment that this is for stage only, and a quick note on why I like it. (hopefully recycling has the implications I'm aiming for).